no-computer.md

```diff
@@ -8,44 +8,99 @@ good way to meet their needs.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
+experience like now? What would their experience be like if you could help
 them?_
 
+In indoor rock climbing, a routesetter will take a variety of plastic climbing
+holds and arrange them on the wall (which contain holes every three to six
+inches) to compose or set a rock climbing route. Climbers then try to ascend the
+route. While routesetters can be extremely creative with how they angle the
+holds and how they combine them, they are limited by the different shaped holds
+they have.
+
+I would like to empower the routesetter with being able to make their own
+holds out of reusable composable pieces to allow for more creativity and tighter
+budgets. Currently, if a routesetter wants a hold that is a different shape from
+what they have, they must purchase it. Additionally, indoor rock climbing would
+become a lot more interesting if there were an increased variety of holds
+reducing the predictability for climbers. Many gyms purchase all their holds
+from the same manufacturer thus frequenters of the gym eventually know exactly
+what they are about to grab without getting to it first. In outdoor climbing,
+you never know how "good" or "bad" a hold is until you get to it which offers
+an extra challenge and more fun!
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+As I mentioned before, if a routesetter does not have a hold in the shape they
+want, they must purchase it specially. Creating a physical DSL would allow
+routesetters to purchase smaller composable pieces that could be combined to
+create holds they envision without having to make extra purchase.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Before arriving at college, I used to competively rock climb and thus spent
+a lot of time in indoor climbing gyms both climbing and setting routes on
+occasion. It would have been a very useful experience to get practice with
+a variety of shapes that aren't pre-determined by the manufacturer since when
+you go and compete at other gyms and competitions, they will use holds you may
+have never encountered before.
+
+I had this idea since I noticed many gyms have to make a huge investment in
+differnt climbing holds when at any given time, only half of which are on the
+wall. I would like to eliminate the need for needing to have this huge stock
+of holds.
```

Comment:
I really like this project in theory, but I worry in practice that people will just go with the standard option since it's less work for them.